tests/sensors/test_external_task_sensor.py

```diff
@@ -307,6 +307,27 @@ def test_external_task_sensor_soft_fail_failed_states_as_skipped(self):
         assert len(task_instances) == 1, "Unexpected number of task instances"
         assert task_instances[0].state == State.SKIPPED, "Unexpected external task state"
 
+    def test_external_task_sensor_skipped_states_as_skipped(self):
+        self.add_time_sensor()
+        op = ExternalTaskSensor(
+            task_id="test_external_task_sensor_check",
+            external_dag_id=TEST_DAG_ID,
+            external_task_id=TEST_TASK_ID,
+            allowed_states=[State.FAILED],
+            skipped_states=[State.SUCCESS],
+            dag=self.dag,
+        )
+
+        # when
+        op.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
+
+        # then
+        session = settings.Session()
```

Comment:
Use the `session` fixture instead please.

Reply:
Ah sorry,  I was just copying an existing test... Will fix.

Reply:
(this is fixed)